### ISSUE TITLE
InfoNCE loss function is not used, however a vanilla cross entropy is used

### ISSUE THREAD

**Issue by arnavsm** [cogtoolslab/photo-sketch-correspondence#5]
Can the authors or someone else provide clarity?
